client/src/nl/sensorlab/videowall/animation/baseanimations/flocking/Flock.java

```diff
@@ -0,0 +1,44 @@
+package nl.sensorlab.videowall.animation.baseanimations.flocking;
+import java.util.ArrayList;
+
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import processing.core.PGraphics;
+
+public class Flock {
+
+	FlockingAnimation parent;
+	
+	public  ArrayList<Boid> boids;
+	public int[] colors;
```

Comment:
If this list is identical to the list of colors in FlockingAnimation (which I think it is), it makes sense to use just that rather than keeping the same list in 13 places.